### PR TITLE
measure resource consumption with Vertical Pod Autoscaler

### PR DESCRIPTION
**Pull request by pohly**
This is useful for replacing the guessed default resource requirements
in the operator.

### CODE REVIEW

**Comment by pohly** [comment]
This was straightforward, but still required a bit of reading of documentation and the TLS issue is a pitfall. Therefore it is worthwhile to record the result in our repo.

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -258,6 +259,40 @@ The klog.Info statements are used via the verbosity checker using the following
...
+
+Resource requirements depend on the workload. To generate some load, run
+```console
+make test_e2e TEST_E2E_FOCUS=lvm-production.*late.binding.*stress.test
+```
+
+Now resource recommendations can be retrieved with:
+
+```console
+kubectl get vpa
```
Comment (avalluri) [inline docs/DEVELOPMENT.md:293]: `kubectl get vpa`  itself does not give any useful information. Would be nice to mention a way to use vpa reccomnedation something like: 
```
$ kubectl get vpa  pmem-csi-node -o jsonpath='{range .status.recommendation.containerRecommendations[*]}{.containerName}{":\n\tRequests: "}{.lowerBound}{"\n\tLimits: "}{.upperBound}{"\n"}{end}'
```

Comment (pohly) [reply docs/DEVELOPMENT.md:293]: Right. I think I was using `kubectl describe` and should have put that into the doc.

Let me add that and the command that you showed here.

Comment (pohly) [reply docs/DEVELOPMENT.md:293]: Done - let's merge if okay without waiting for tests.

**Review by avalluri — APPROVED** [review]

**Pull request merged**